go/kbfs/libkbfs/conflict_resolver.go

```diff
@@ -3335,7 +3334,7 @@ func (cr *ConflictResolver) doResolve(ctx context.Context, ci conflictInput) {
 		cr.log.CWarningf(ctx,
 			"Could not record conflict resolution attempt: %v", err)
 	} else {
-		defer cr.recordFinishResolve(ctx, ci, err)
+		defer func() { cr.recordFinishResolve(ctx, ci, err) }()
```

Comment:
Ooh good call.  This change should probably go in the original PR if you haven't merged it yet.

Reply:
I was waiting to merge it in case this PR caused it changes.